R/Rmath/build_tarballs.jl

```diff
@@ -11,7 +11,7 @@ sources = [
 script = raw"""
 cd $WORKSPACE/srcdir/Rmath-julia
```

Comment:
I think we now need to enter `src`?
```suggestion
cd $WORKSPACE/srcdir/Rmath-julia/src
```